user/serializers.py

```diff
@@ -22,3 +23,37 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
```

Comment:
I think ObtainAuthToken's default serializer would be enough but its up to you, you can leave like this